libs/localization/assets/locales/en/page_character_optimize.json

```diff
@@ -22,29 +22,29 @@
     "buildAlreadyInList": "Build is already in the list"
   },
   "levelFilter": "Artifact Level Filter",
-  "useExcluded": {
-    "title": "Use Excluded Artifacts",
+  "excludeArt": {
+    "title_exclude": "Exclude Artifacts",
+    "title_tooltip": "Add artifacts to this character's exclusion list. Artifacts in the exclusion list will not be considered for build generation.",
     "usingNum_one": "Using <1>{{count}}</1> excluded artifacts",
     "usingNum_other": "Using <1>{{count}}</1> excluded artifacts",
     "excNum_one": "<1>{{count}}</1> artifact is excluded",
-    "excNum_other": "<1>{{count}}</1> artifacts are excluded"
+    "excNum_other": "<1>{{count}}</1> artifacts are excluded",
+    "addExc": "Add artifact to this character's exclusion list",
+    "selExc": "Select artifact to add to this character's exclusion list",
+    "deSelectExc": "Click on an artifact to remove it from the exclusion list",
+    "excludeArtifactTip": "Excluding an artifact will prevent the build generator from using it for this character.",
+    "excluded": "This artifact is excluded.",
+    "included": "This artifact is not excluded."
   },
-  "useEquipped": {
-    "title": "Use Equipped Artifacts",
+  "excludeChar": {
+    "title": "Allow Equipped Artifacts",
     "modal": {
-      "title": "Character Priority for Equipped Artifacts",
-      "desc1": "When generating a build, the Optimizer will only consider equipped artifacts from characters below the current character or those not on the list.",
-      "desc2": "If the current character is not on the list, the Optimizer will only consider equipped artifacts from others characters that are not on the list.",
-      "usingNum_one": "Using artifacts from <1>{{count}}</1> character",
-      "usingNum_other": "Using artifacts from <1>{{count}}</1> unlisted characters",
-      "status": {
-        "use": "Use artifacts",
-        "dont": "Don't use artifacts",
-        "curr": "Current character"
-      },
-      "add": "Add character to list",
-      "clearList": "Clear List"
+      "title_sel": "Allow equipped artifacts",
```

Comment:
```suggestion
```
Remove this string and just use "excludeChar.title" for the title of the modal

Reply:
I think the way we have been exporting translations is wrong, since it would include fuzzy, which means that there will be translations that hasn't been updated. I've updated the exports in POEditor to "not fuzzy", usually I elect to just use a different key

Reply:
I dont understand what youre saying or how that is relevant to this comment